src/browser/atom-window.coffee

```diff
@@ -73,6 +73,9 @@ class AtomWindow
     @browserWindow.loadUrl @getUrl(loadSettings)
     @browserWindow.focusOnWebView() if @isSpec
 
+    if locationsToOpen.length is 1 and locationsToOpen[0].pathToOpen is null
```

Comment:
Yeah, so in this case, wouldn't you want it to go down that path if it is also `undefined`?